ggml/src/ggml-backend.cpp

```diff
@@ -126,6 +126,7 @@ void * ggml_backend_buffer_get_base(ggml_backend_buffer_t buffer) {
     return base;
 }
 
+// Check with reviewers: any cons for that method to return a ggml_status?
```

Comment:
This function must return a `ggml_status` otherwise it would be impossible to take advantage of this change.

Reply:
ok done